docs/samples/extension/README.md

```diff
@@ -0,0 +1,58 @@
+This sample shows how to extend the tekton dashboard using the dashboard extension.  The extensions should be in experimental repository (https://github.com/tektoncd/experimental)
```

Comment:
> This sample shows how to extend the tekton dashboard using the dashboard extension.

"This sample shows how to extend the tekton dashboard using the extension mechanism" might be better. 

> The extensions should be in experimental repository (https://github.com/tektoncd/experimental)

Perhaps we could say, 'This sample extension is intended just to illustrate the mechanism. See the 'experimental' repository (https://github.com/tektoncd/experimental) for more extensions.'